### PR TITLE
[Mar 3680] Implement RemovePolicies and AddPolicies for Casbin 2.2.2

### PR DESCRIPTION
**Pull request by dallen42**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -15,6 +15,8 @@ github.com/apache/thrift v0.12.0/go.mod h1:cp2SuWMxlEZw2r+iP2GNCdIi4C1qmUzdZFSVb
 github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973/go.mod h1:Dwedo/Wpr24TaqPxmxbtue+5NUziq4I4S80YR8gNf3Q=
 github.com/casbin/casbin/v2 v2.0.0 h1:OIcnP8SxwF1gmGxOn7Kod/O/7yJikpHWQz0qiBJpG/U=
```
Comment (hsluoyz) [inline go.sum]: Please remove Casbin v2.0.0.

**Comment by hsluoyz** [comment]
RemovePolicies and AddPolicies features are very good. Are you willing to contribute it back to: https://github.com/casbin/gorm-adapter by making a PR ?

File: go.sum (hunk 1 above)
Comment (dallen42) [reply go.sum]: Done, good catch

**Comment by dallen42** [comment]
@hsluoyz yes, was planning on submitting a PR once I did some testing with our code base

**Comment by hsluoyz** [comment]
Thanks!

**Pull request merged**